src/sklearn_evaluation/plot/__init__.py

```diff
@@ -10,7 +10,7 @@
 )
 
 from .roc import roc, ROC
-from .precision_recall import precision_recall
+from .precision_recall import PrecisionRecall, precision_recall
```

Comment:
Let's avoid relative imports

Reply:
I didn't understand. All the plot imports need to be changed or only precision recall?

Reply:
Only the ones that you're touching in this PR

Reply:
Clarification: Change all of the imports in those classes.